src/js/components/SidebarFilter.js

```diff
@@ -136,12 +138,17 @@ class SidebarFilter extends mixin(QueryParamsMixin) {
   }
 
   getHealthCheckboxes() {
-    let {filterLabels, filterValues} = this.props;
+    let {filterLabels, filterType, filterValues} = this.props;
     let {selectedNodes} = this.state;
 
     return Object.keys(filterLabels)
       .filter(function (filterLabel) {
```

Comment:
Because of the this in line 151 you want to change this line to
`(filterLabel) => {` to keep the context.